src/terraform/install.sh

```diff
@@ -26,9 +26,11 @@ TERRAFORM_DOCS_SHA256="${TERRAFORM_DOCS_SHA256:-"automatic"}"
 
 TERRAFORM_GPG_KEY="72D7468F"
 TFLINT_GPG_KEY_URI="https://raw.githubusercontent.com/terraform-linters/tflint/master/8CE69160EB3F2FE9.key"
```

Comment:
Can you bump the minor version? 
https://github.com/devcontainers/features/blob/ea03d367f118b11b2a85149781dc16083d977e5e/src/terraform/devcontainer-feature.json#L3

Reply:
yes. i bump up miner version.
Maybe tomorrow.

Reply:
Increment minor version.

Reply:
Incorporated upstream changes and fixed from 1.2.0 to 1.2.1.

Reply:
Could you please confirm the change?